app/src/components/guestbook/GuestBookMessageEdit.tsx

```diff
@@ -34,45 +41,93 @@ export default function GuestBookMessageEdit({
         []
     );
 
-    const updateMessage = useCallback(
-        (newValues: Partial<IGuestBookMessage>) => {
-            setMessage({
-                ...(message as IGuestBookMessage),
-                ...newValues,
-            });
-        },
-        [message, setMessage]
-    );
+
+    const initializeValues = (): NewGuestBookMessage | IGuestBookMessage => {
+        if (isUndefined(message)){
+            return getInitialMessage()
+        }
+        return message!
```

Comment:
Can you make sure these lines end with semicolons `;`? If you install the Pretttier extension in your IDE, your code will be automatically formatted correctly :)